WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations-2004.md

```diff
@@ -0,0 +1,794 @@
+
+|Name|Comment|
+|---|--------|
+|Cellcore|[https://docs.microsoft.com/en-us/windows-hardware/drivers/network/cellular-architecture-and-driver-model](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/cellular-architecture-and-driver-model)|
+|CloudExperienceHostOOBE|Documented [here](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-how-it-works-technology#cloud-experience-host).|
+|DiagLog|A log generated by the Diagnostic Policy Service, which is documented [here](https://docs.microsoft.com/en-us/windows-server/security/windows-services/security-guidelines-for-disabling-system-services-in-windows-server)|
+|RadioMgr|https://docs.microsoft.com/en-us/windows-hardware/drivers/nfc/what-s-new-in-nfc-device-drivers|
+|ReadyBoot|Documentation [here](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/xperf/readyboot-analysis).|
+|WDIContextLog|Wireless Local Area Network (WLAN) Device Driver Interface, and is documented here. |
+|WiFiDriverIHVSession|Documented [here](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/user-initiated-feedback-normal-mode).|
+|WiFiSession|Diagnostic log for WLAN technology. If Wi-Fi not implemented, no need for this logger|
+|WinPhoneCritical|Diagnostic log for phone (Windows?). If not using phones, no need for this logger|
+
+#### Windows Defender optimization in the virtual desktop environment
+Microsoft has recently published documentation regarding Windows Defender in a virtual desktop environment, at this location:
+
+[Deployment guide for Windows Defender Antivirus in a virtual desktop infrastructure (VDI) environment](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/deployment-vdi-windows-defender-antivirus)
+
+The above article contains procedures to service the ‘gold’ virtual desktop image, and how to maintain the virtual desktop clients as they are running. To reduce network bandwidth when virtual desktop devices need to update their Windows Defender signatures, stagger reboots, and schedule reboots during off hours where possible. The Windows Defender signature updates can be contained internally on file shares, and where practical, have those files shares on the same or close networking segments as the virtual desktop devices.
+
+#### Client network performance tuning by registry settings
+There are some registry settings that can increase network performance. This is especially important in environments where the virtual desktop device or physical computer has a workload that is primarily network based.  The settings in this section are recommended to tune performance for the networking workload profile, by setting up additional buffering and caching of things like directory entries and so on.
+
+> [!NOTE]Some settings in this section are registry-based only and should be incorporated in the base image before the image is deployed for production use.
```

Comment:
Please use the recommended MD Note blob codestyle:
```suggestion
> [!NOTE]
> Some settings in this section are registry-based only and should be incorporated in the base image before the image is deployed for production use.
```

Reply:
Fixed